AlphaWallet/Transfer/ViewModels/ConfirmSignMessageViewControllerViewModel.swift

```diff
@@ -258,7 +258,7 @@ extension EIP712TypedData.JSON {
             return NSAttributedString(string: fittedString.indented(indentationLevel), attributes: valueAttributes)
 
         case .number(let value):
-            return NSAttributedString(string: String(value).indented(indentationLevel), attributes: valueAttributes)
+            return NSAttributedString(string: value.debugDescription.indented(indentationLevel), attributes: valueAttributes)
```

Comment:
The PR fixes the issue, but can you replace `debugDescription` with something else? The result returned by `debugDescription` is subject to change by Apple.